packages/gluestick/src/cli/index.js

```diff
@@ -205,6 +205,23 @@ commander
     }),
   );
 
+// This command is used by the global `gluestick-cli` package CLI to append
+// the commands listed in this file to its help message.
+commander.command('print-help', null, { noHelp: true }).action(() => {
+  commander.help(txt => {
+    // Remove the first 11 lines because the global `gluestick-cli` package CLI
+    // prints similar information. Note: `txt` passed to `.help()` uses '\n' to
+    // separate lines.
+    // https://github.com/tj/commander.js/blob/d8404f8de45c9ba780606878f4d35d0a45743d32/index.js#L1130
+    return txt
+      .split('\n')
+      .slice(11)
```

Comment:
If we add another command in `gluestick-cli`, would we need to update this number?

Reply:
That's a good point-- adding new commands in `gluestick-cli` or `gluestick` CLIs should not affect this number, but adding a new option in the `gluestick` CLI would. I wanted to completely override the default `--help` message, but `commander` didn't support this AFAICT.